core/test/common/test_config.h

```diff
@@ -26,6 +26,31 @@
  *
  */
 
-#define POSTGRES_ON_LOCALHOST "postgres://localhost:5432"
-#define POSTGRES_TEST_DB "test_db"
-#define POSTGRES_TEST_DB_ON_LOCALHOST POSTGRES_ON_LOCALHOST "/" POSTGRES_TEST_DB
+#include <cstdlib>
+#include <string>
+
+inline std::string getPostgresUrl() {
+    const char *host = getenv("POSTGRES_HOST");
+    std::string host_var(host ? host : "localhost");
```

Comment:
`const` are missing (except for `user_var`)

Reply:
Right !